com.smallstep.Smallstep.yml

```diff
@@ -0,0 +1,101 @@
+id: com.smallstep.Smallstep
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: smallstep
+rename-icon: smallstep
+
+finish-args:
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf # For GNOME proxy resolution
+  - --env=GIO_EXTRA_MODULES=/app/lib/gio/modules # For GNOME proxy resolution
+  - --env=GSETTINGS_BACKEND=dconf # For GNOME proxy resolution
+  - --filesystem=~/.config/dconf:ro # For GNOME proxy resolution
+  - --filesystem=~/.config/kioslaverc # For KDE proxy resolution (KDE5 only)
+  - --filesystem=xdg-run/dconf # For GNOME proxy resolution
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=system-bus
+  - --socket=session-bus
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --system-talk-name=org.freedesktop.NetworkManager
+  - --device=all
+
+modules:
```

Comment:
I remove the `pcsclite` module, it's not needed in production builds and those will not be compiled with YubiKey support.